lambdas/postgres-migration-async-operation/variables.tf

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
Nit sort by required/not required and consider alpha sorting. 

😅  I say this with full acknowledgement that my most recent PR may *not* have done this.